src/Data/Rational/Unnormalised/Properties.agda

```diff
@@ -107,6 +126,19 @@ p ≃? q = Dec.map′ *≡* drop-*≡* (↥ p ℤ.* ↧ q ℤ.≟ ↥ q ℤ.* 
   ; _≟_           = _≃?_
   }
 
+≠-isApartnessRelation : IsApartnessRelation _≃_ _≠_
+≠-isApartnessRelation = record
+  { irrefl  = ≃-≠-irreflexive
+  ; sym     = ≠-symmetric
+  ; cotrans = ≠-cotransitive
+  }
+
+≠-tight : Tight _≃_ _≠_
+proj₁ (≠-tight p q) ¬p≠q with p ≃? q
+... | yes p≃q = p≃q
+... | no ¬p≃q = ⊥-elim (¬p≠q ¬p≃q)
```

Comment:
Can we use `contradiction` instead?

Reply:
I think it would be even better to use `decidable-stable` here.

Reply:
It is using decidable-stable instead.